### PR TITLE
Basic logic to read from serial

### PR DESCRIPTION
**Pull request by agrymaloski**
# Description

Fixes # (issue)

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### CODE REVIEW

File: src/app.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 """ Main loop that is run on the Raspberry Pi on the USV 🛳⚓️ """
-
+import configparser
```
Comment (zthh626) [inline src/app.py:2]: Why do we import `configparser` if we don't end up using it?

File: src/serial/serial.py [hunk 2]
```diff
@@ -1,13 +1,27 @@
...
+from os import system
 import time
+from serial import Serial
 from typing import List, Tuple, Any
 from src.events.event_type import EventType
+import json
 
-
-def serial_loop(event_list: List[Tuple[EventType, Any]]):
+def serial_loop(event_list: List[Tuple[EventType, Any]], serial : Serial):
```
Comment (zthh626) [inline src/serial/serial.py:9]: Is there a reason we import `serial` when we make it within this function?

**Review by zthh626 — CHANGES_REQUESTED** [review]

**Review by zthh626 — APPROVED** [review]

File: src/app.py [hunk 3]
```diff
@@ -11,32 +10,25 @@
...
         setup_database_handlers(manager, subscribers)
         setup_serial_handlers(manager, subscribers)
         setup_socketio_handlers(manager, subscribers)
-
         processes["driver"] = Process(target=driver_loop, args=(event_list,))
-        processes["serial"] = Process(target=serial_loop, args=(event_list,))
-        processes["event_loop"] = Process(
-            target=run_event_loop, args=(subscribers, event_list)
-        )
+        processes["serial"] = Process(target=serial_loop,  args=(event_list))
```
Comment (zthh626) [inline src/app.py:30]: args has to be a tuple, `args=(event_list,)`, you need the comma

**Review by zthh626 — CHANGES_REQUESTED** [review]

**Comment by agrymaloski** [comment]
trying to merge

**Review by zthh626 — APPROVED** [review]

**Pull request merged**